providers/base/units/power-management/jobs.pxu

```diff
@@ -134,6 +134,53 @@ _description:
  VERIFICATION:
      Did the system resume when the lid was opened?
 
+plugin: user-interact-verify
+category_id: com.canonical.plainbox::power-management
+id: power-management/lid_close_suspend_open
+estimated_duration: 20.0
+requires: device.product == 'Lid Switch'
+_purpose:
+    This test will check your lid sensor can detect lid close/open, and DUT will suspend when the lid closed
+_steps:
+    1. Press "Enter" to start the test
+    2. Close the lid (Plese close the lid within 10 sec)
```

Comment:
Small typo here: `Plese` should be `please`

Reply:
Fix the typo~